packages/help-center/src/components/help-center-header.tsx

```diff
@@ -82,6 +82,7 @@ const HelpCenterHeader = ( { isMinimized = false, onMinimize, onMaximize, onDism
 							<Route path="/inline-chat">{ __( 'Live Chat', __i18n_text_domain__ ) }</Route>
 							<Route path="/contact-form" component={ SupportModeTitle }></Route>
 							<Route path="/post" component={ ArticleTitle }></Route>
+							<Route path="/success">{ __( 'Messagge submitted', __i18n_text_domain__ ) }</Route>
```

Comment:
```suggestion
							<Route path="/success">{ __( 'Message Submitted', __i18n_text_domain__ ) }</Route>
```

Got an extra G in there. Besides that everything looks good 👍 tested and it worked well.

Reply:
Thanks for the catch, updated 👍